src/main/resources/templates/about-us.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>About Us</title>
+</head>
+<body>
+    <h2>About Us</h2>
```

Comment:
Suggstion: add Bootstrap for all kinds of buttons, tables, columns. I don"t know a better framework for such things:
https://getbootstrap.com/docs/5.3/getting-started/introduction/

Reply:
have to say, its more about front-end thing, and we should abandon such form thing in the future, maybe

Reply:
I removed this file completely

Reply:
@Miao-99 we need some sort of visual overviews anyway. 
@JannesVolkens we will need those views later

Add this to pom file

<dependency>
			<groupId>org.webjars</groupId>
			<artifactId>bootstrap</artifactId>
			<version>5.2.3</version>
		</dependency>
		<dependency>
			<groupId>org.webjars</groupId>
			<artifactId>jquery</artifactId>
			<version>3.6.1</version>
		</dependency>

Reply:
```
<dependency>
			<groupId>org.webjars</groupId>
			<artifactId>bootstrap</artifactId>
			<version>5.2.3</version>
		</dependency>
		<dependency>
			<groupId>org.webjars</groupId>
			<artifactId>jquery</artifactId>
			<version>3.6.1</version>
		</dependency>
```

Reply:
Done